test/Altinn.Platform.Authentication.Tests/Mocks/SystemUserServiceMock.cs

```diff
@@ -190,4 +171,9 @@ public Task<bool> SetDeleteFlagOnSystemUser(string partyId, Guid systemUserId, C
         toBeDeleted.IsDeleted = true;
         return Task.FromResult(true);
     }
+
+    Task<Result<SystemUser>> ISystemUserService.CreateSystemUser(string party, SystemUserRequestDto request, int userId)
```

Comment:
Are we still using these mocks? Our tests are now integrated with db right. Time to remove this?

Reply:
yes, but tried to keep this PR small.
will create a new housekeeping PR